src/index.js

```diff
@@ -1,14 +1,102 @@
-import _ from 'lodash';
 import './style.css';
+import Tasks from './tasks.js';
+import statusUpdateBtns from './status.js';
 
-function component() {
-  const element = document.createElement('div');
+const toDoTasks = new Tasks();
+const listContent = document.querySelector('.list-content');
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
-  
-  return element;
-}
+// populate list items
+const populateListItems = (arrayOfTasks) => {
+  listContent.innerHTML = '';
+  arrayOfTasks.forEach((task) => {
+    listContent.innerHTML += `
+    <li>
+      <button class="check-btn" ><i class="fa-solid fa-check ${
+  task.completed ? 'active' : ''
```

Comment:
I think it would have been better to use the checkbox input type here. It works better than a button😄